src/audio/wav64_opus.c

```diff
@@ -54,29 +55,29 @@ typedef struct {
 /// @brief Wav64 Opus state
 typedef struct {
     wav64_opus_header_ext xhead;    ///< Opus header extension
-    uint32_t current_rom_addr;      ///< Current ROM reading address
+    int current_fd;                 ///< Current file descriptor for the wav64
```

Comment:
Given that the code below always have a pointer to the wav64 and you have the file descriptor there, I think you don't need this variable here. There is only one object (the wav64), only one file descriptor, and only one current position being read.